omni/omni.py

```
@@ -0,0 +1,209 @@
+            moderated_users=[],
+            category_counter={}
+        )
+        self.client = OpenAI()
+        self.message_count = 0
+        self.moderated_count = 0
+        self.moderated_users = set()
+        self.category_counter = Counter()
+
+    async def initialize(self):
+        all_guilds = await self.config.all_guilds()
+        for guild_id, data in all_guilds.items():
```

Ruff (0.8.2)

Loop control variable `guild_id` not used within loop body

Rename unused `guild_id` to `_guild_id`

(B007)